esrally/client/asynchronous.py

```diff
@@ -223,7 +223,7 @@ def __init__(self, config):
         self._loop = None
         self.client_id = None
         self.trace_configs = None
-        self.enable_cleanup_closed = None
+        self.enable_cleanup_closed = False
```

Comment:
@favilo This is the only material change in https://github.com/elastic/rally/pull/1859/commits/13f1272b4e7d713a73c3fc27c9b79aa877b4b0cd I think. I've followed base classes and confirmed `enable_cleanup_closed` defaults to `False`, please check.

Reply:
Yeah, that is why I chose `False`, because it's the default in the base class.